sidebars/core.js

```diff
@@ -5,7 +5,7 @@ const Sidebar = [
   {
     extra: {
       classNames: ['label'],
-      icon: 'img/core-contributors.svg',
+      icon: 'img/core-contributors-sm.svg',
```

Comment:
? these are all scalable

Reply:
They are cropped differently between small and medium. Do you want me to ask scott to resave them? Then we could just use one

Reply:
If we can use 1 and apply styles that would be much better.